REST_API_DOCS.md

```
@@ -0,0 +1,1623 @@
+    "message": "이미 존재하는 데이터가 있습니다.",
+    "details": [
+      {
+        "field": "email",
+        "issue": "이미 가입된 이메일 주소입니다."
+      }
+    ]
+  }
+}
+```
+
+**Error Response (400 BAD_REQUEST)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)